translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/FilesPage/FeedbackDialog.java

```diff
@@ -0,0 +1,69 @@
+package org.wycliffeassociates.translationrecorder.FilesPage;
+
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import org.wycliffeassociates.translationrecorder.R;
+import com.door43.tools.reporting.Logger;
+
+import java.io.File;
+
+public class FeedbackDialog extends DialogFragment implements View.OnClickListener {
```

Comment:
rotating the screen will likely kill this dialog. I don't think this would have an impact on app stability, but is it acceptable behavior?

Reply:
Good point @sarabiaj , I didn't think of it. I'll take a look.

Reply:
@sarabiaj I checked, it doesn't crash, only message gets empty, but progress dialog crashes, when screen is  locked and unlocked again, but on screen rotation everything is ok.

Reply:
when you say "progress dialog crashes" do you mean that it just goes away, or the app crashes?

Reply:
sorry, yes the app crashes